contents/handbook/people/team-structure/infrastructure.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Team Infrastructure
+sidebar: Handbook
+showTitle: true
+hideAnchor: true
+---
+
+## People
+
+- James Greenhill (Team lead, Data/Infra Engineer)
+- Guido Iaquinti (Site Reliability Engineer)
+- Harry Waye (Full Stack Engineer)
+
+## Mission Statement
+Make deploying, scaling, and managing PostHog easy, fast, and reliable.
+
+## Q3 2022 Goals
+* **Proposed Objective**:
+    * Lots of customers are self-serving PostHog Cloud in the EU
+* **Proposed Key Results**:
+    * >100 organizations using EU hosted PostHog Cloud
```

Comment:
Having an org count target is out of control for the team. 

```suggestion
    * EU hosted PostHog Cloud is available at scale (to meet demand)
```

Reply:
@camerondeleone and @simfish85 will be able to make getting organizations onto EU cloud easy (I'm sure they have a long list of potential clients).

Perhaps the number is too big, but I think it's only really done when real people are using it - We often find out a bunch of problems only when customers are using something in anger.

We could
```suggestion
    * Multiple organizations using EU hosted PostHog Cloud with similar reliability to US cloud now
```